test/apiv2/README.md

```diff
@@ -52,9 +52,14 @@ Notes:
 If there's no leading slash, `t` prepends `/v1.40`. This is a simple
 convenience for simplicity of writing tests.
 
-* When method is POST, the argument after the endpoint must be a series
-of POST arguments in the form 'key=value', separated by commas. `t` will
-convert those to JSON form for passing to the server.
+* When method is POST, the argument(s) after the endpoint may be a series
+of POST parameters in the form 'key=value', separated by spaces:
+     t POST myentrypoint 200                                 ! no params
+     t POST myentrypoint id=$id 200                          ! just one
+     t POST myentrypoint id=$id filter='{"foo":"bar"}' 200   ! two, with json
+     t POST myentrypoint name=$name badparam='["foo","bar"]' 500  ! etc...
```

Comment:
Would `foo="value with space"` be counted as one argument?

Reply:
Oh, good question. I was careful to do arg processing the "correct" way, so yes, it should work, but I don't have any explicit tests for it. I think it might be time to add an internal test suite for that helper. Unfortunately, my sneak-in-some-work-time for today has run out! I'll get to that when I can. Thanks for pointing it out.

Reply:
Thank you, LGTM :)